packages/contracts-bedrock/src/governance/ProposalValidator.sol

```diff
@@ -11,11 +11,16 @@ import { Predeploys } from "src/libraries/Predeploys.sol";
 import { IOptimismGovernor } from "interfaces/governance/IOptimismGovernor.sol";
 import { IGovernanceToken } from "interfaces/governance/IGovernanceToken.sol";
 import { IEAS, Attestation } from "src/vendor/eas/IEAS.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
+// Modules
+import { ProposalSettings, ProposalOption, PassingCriteria } from "src/governance/ApprovalVotingModule.sol";
+
+/// @custom:upgradeable
 /// @title ProposalValidator
 /// @notice The ProposalValidator contract is responsible for validating proposals and moving
 ///         them to the vote phase on the Optimism Governor.
-contract ProposalValidator is Ownable {
+contract ProposalValidator is OwnableUpgradeable {
```

Comment:
i thought about this, maybe we should follow what OP is doing with their other contracts and use `ReinitializableBase` and `ISemver`